src/services/TestDataService.ts

```diff
@@ -1749,7 +1781,6 @@ export class TestDataService {
         const totalPrice = unitPrice * (lineItem.quantity || 1);
 
         const basicPromotionLineItemStruct = {
-            promotionId: promotion.id,
```

Comment:
Why do you delete that `promotionId` field?

Reply:
it's not allowed to provide the id in 6.5 for some reason

Reply:
I'll double check this, because I'm not entirely sure how it works

Reply:
@pweyck I had trouble to create a Promotion via AdminAPI and ask Soner would could be the problem. The solution was: https://gitlab.shopware.com/shopware/6/product/platform/-/commit/bc1be7402fff6104adf71194c056d240b640192d

Reply:
The solution is to backport the MR from 6.6. to 6.5.

Reply:
I've backported the fix and will revert this change